app/adapters/application.js

```diff
@@ -1,2 +1,40 @@
-import DS from "ember-data";
-export default DS.FixtureAdapter.extend({});
\ No newline at end of file
+import Ember from 'ember';
+import config from '../config/environment';
+import PouchDB from 'pouchdb';
+import { Adapter } from 'ember-pouch';
+
+var db = new PouchDB(config.local_couch || 'bloggr');
+var remote = new PouchDB(config.remote_couch, {ajax: {timeout: 20000}});
+
+db.sync(remote, {live: true, retry: true});
+
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'destructuring expression' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).